lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:cboard_mobile/Settings/ThemeManager.dart';
+import 'package:cboard_mobile/Settings/routes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'routes.dart';
```

Comment:
Just curious, but what does the `ThemeManager` and `rendering` packages do?

Reply:
Nvm that. It's for incomplete dark mode. I'll comment it then.